### PR TITLE
Remove macOS 10.14 from buildkite pipeline

### PR DESCRIPTION
**Pull request by bogniq**
## Change Description

macOS 10.14 support is out of date and thus removed from CI pipeline for `develop-boxed` branch. (It has already been removed from `develop` and `release/1.8.x`, see https://github.com/EOSIO/eosio.cdt/pull/1243)

## API Changes
- [ ] API Changes

## Documentation Additions
- [ ] Documentation Additions

### CODE REVIEW

**Review by softprofe — COMMENTED** [review]
Is there no .cicd/platforms/macos-10.14.sh in develop-boxed?

**Comment by bogniq** [comment]
@softprofe Good catch, now "macos-10.14.sh" is removed.

**Review by softprofe — APPROVED** [review]

**Review by kj4ezj — CHANGES_REQUESTED** [review]
Mojave is still in the pipeline documentation.

Screen Shot 2022-01-24 at 10 12 30 AM

**Comment by bogniq** [comment]
@kj4ezj Good catch, fixed.

**Review by kj4ezj — CHANGES_REQUESTED** [review]
I hate to kick it back again, but there is still Mojave code.

Screen Shot 2022-01-25 at 11 30 00 AM

If it helps, I am just using `grep` to find these.
```bash
grep --exclude-dir .git -rni '10.14'
grep --exclude-dir .git -rni 'mojave'
```

**Comment by bogniq** [comment]
@kj4ezj @softprofe I changed it to be consistent with current https://github.com/EOSIO/eos/blob/develop/scripts/generate_bottle.sh, but please let me know if there are better fix for it.

File: scripts/generate_bottle.sh [hunk 1]
```diff
@@ -1,15 +1,15 @@
 #! /bin/bash
 
-VERS=`sw_vers -productVersion | awk '/10\.14\..*/{print $0}'`
+VERS=`sw_vers -productVersion | awk '/10\.15\..*/{print $0}'`
 if [[ -z "$VERS" ]]; then
-   VERS=`sw_vers -productVersion | awk '/10\.15.*/{print $0}'`
+   VERS=`sw_vers -productVersion | awk '/11\.*.*/{print $0}'`
```
Comment (softprofe) [inline scripts/generate_bottle.sh]: Could you test this part ?       awk in line 5 looks a * more than required, not sure it will work you need test it.

Comment (bogniq) [reply scripts/generate_bottle.sh]: My bad, it's a typo. Just fixed.

**Review by kj4ezj — APPROVED** [review]
🍎

**Pull request merged**